IFC4x3/PropertyConstants/c/COAXIAL_0EDF1tE4f7IRLmNK3xztkj.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="COAXIAL_0EDF1tE4f7IRLmNK3xztkj" Name="COAXIAL" UniqueId="0e34f077-384a-4749-b570-5d40fbf77bad" />
+
```

Comment:
Document if not self-evident (even though even I know this one)

Reply:
added